src/mentors.json

```diff
@@ -2710,5 +2177,28 @@
         "id": "_gmaiolo"
       }
     ]
+  },
+  {
+    "id": "alejonanez@gmail.com",
```

Comment:
This is the _relevant_ part of this PR.

Reply:
Try to remove those changes :)

Reply:
Done in f03aa1573a250ebbbe02fd82cbc770b666b08994